python/statsforecast/core.py

```diff
@@ -1,3 +1,5 @@
+"""Methods for Fit, Predict, Forecast (fast), Cross Validation and plotting"""
+
```

Comment:
```suggestion
```

Reply:
This should fix the linter error, I don't have permission to push to your branch.